internal/civisibility/integrations/gotesting/testingB.go

```diff
@@ -33,6 +28,12 @@ var (
 
 	// subBenchmarkAutoNameRegex is a regex pattern to match the sub-benchmark auto name.
 	subBenchmarkAutoNameRegex = regexp.MustCompile(`(?si)\/\[DD:TestVisibility\].*`)
+
+	// civisibilityBenchmarksFuncs holds a map of *func(*testing.B) for tracking instrumented functions
+	civisibilityBenchmarksFuncs = map[*func(*testing.B)]bool{}
```

Comment:
nit: I think using a `map[*func(*testing.B)]struct{}` could be more efficient as a hashset datastructure?

Reply:
done in [6a4ca88](https://github.com/DataDog/dd-trace-go/pull/2843/commits/6a4ca887a52919f7f3673b33f19aafb2356dbe65)